integration-libs/epd-visualization/.release-it.json

```diff
@@ -34,7 +34,6 @@
             "peerDependencies.@spartacus/core",
             "peerDependencies.@spartacus/styles",
             "peerDependencies.@spartacus/storefront",
-            "peerDependencies.@spartacus/assets",
```

Comment:
Nice. 
I believe we need to repeat the same change in develop, to avoid similar problem when we'll release 5.0 out from develop branch: https://github.com/SAP/spartacus/blob/21a8ce8767ae622015c1ffbecd9912172b7045d5/integration-libs/epd-visualization/.release-it.json#L37

Reply:
PR for change on develop: https://github.com/SAP/spartacus/pull/15260